api/app/Generators/ApplicationDocGenerator.php

```diff
@@ -88,23 +103,7 @@ public function generate(): self
             });
         }
 
-        if (isset($snapshot['experiences'])) {
-            $snapshotExperiences = $snapshot['experiences'];
-
-            // the snapshot stores the department and classification models connected by relation
-            // to render with GeneratesUserDoc, map the model to a string with the appropriate property name
-            foreach ($snapshotExperiences as &$experience) {
-                if ($experience['__typename'] === 'WorkExperience') {
-                    if (isset($experience['department'])) {
-                        $experience['departmentId'] = $experience['department']['id'];
-                    }
-                    if (isset($experience['classification'])) {
-                        $experience['classificationId'] = $experience['classification']['id'];
-                    }
-                }
-            }
-
-            $experiences = Experience::hydrateSnapshot($snapshotExperiences);
```

Comment:
`Experience::hydrateSnapshot($snapshotExperiences)`
Need only be called once and reused, rather than twice like it is currently